lib/bindings/python/rust/lib.rs

```diff
@@ -262,7 +269,11 @@ impl Endpoint {
     }
 
     fn lease_id(&self) -> i64 {
-        self.inner.drt().primary_lease().id()
+        self.inner
+            .drt()
+            .primary_lease()
+            .map(|l| l.id())
+            .unwrap_or(0)
```

Comment:
Does this mean for static worker the lease id will be 0? What about the cases where we have multiple workers in static configuration? Would be lease_id be 0 for all these multiple workers?
Or we can not have multiple workers in static mode?

Reply:
> Or we can not have multiple workers in static mode?

This. The name of the static worker is the namespace/component/endpoint it sits on, so there can only be one.